#22 - checkin/checkout function

OmarRoBra opened the pull request:
# Description
What did you implemented/Why did you implemented this:
 - Controllers and routes of logIn and logOut created.
 - Alerts and notice added to the login page.
 - Added functionality to the login page.

  
  ## Screenshots 
If applicable, include screenshots of the results or screenshots that help to see changes

![image](https://user-images.githubusercontent.com/50344283/127079913-8909f10e-cef5-4b45-b2f2-ed45ed601e1c.png)


![image](https://user-images.githubusercontent.com/50344283/127079859-9046db63-773a-4efa-bcc6-cbfa47eaeb3f.png)


![image](https://user-images.githubusercontent.com/50344283/127079825-257c55a1-23f1-475f-bfde-81338f55e757.png)

OmarRoBra committed:
xd

OmarRoBra committed:
xd

OmarRoBra committed:
!

martinmt-mx reviewed (dismissed):
nice!

YilKanda commented on kata06/app/views/employees/login.html.erb:
```diff
@@ -1,12 +1,20 @@
+  </div>
+<% end %>
 </div>
+<% end %>
```
Add a blank line.

YilKanda commented on kata06/app/controllers/employees_controller.rb:
```diff
@@ -41,4 +72,11 @@ def employee_params
+    
+
+
 end
```
Add a blank line.

YilKanda commented on kata06/config/database.yml:
```diff
@@ -17,6 +17,9 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  host: localhost
+  username: postgres
+  password: alceconbotas2
```
It is not recommended to leave this configuration in Git, what you can do is add this file (database.yml) and exclude it in Git through the .gitignore, and later add a database.yml.example file with the access configuration example to the database.

This is recommended, to the different configurations that each user has, on their computer, of PostgreSQL.

OmarRoBra replied:
Ok, i will do the changes, thanks for the observations :D

YilKanda reviewed (changes requested):
Check the indentation of your *.html.erb and *.rb files (the latter with Rubocop), so that you can have the indentation of two spaces in your files.

martinmt-mx committed:
Unresolved conversations #!

A blank line was added.

martinmt-mx committed:
Unresolved conversations #2

Se agregó una línea en blanco.

OmarRoBra committed:
Fixing issues
We fixed some issues of the pull request like some identatation and we added to the giignore file the path of the database configuration
We did this to have a good pull request

OmarRoBra committed:
Merge branch '22-checkin/checkout_function' of https://github.com/BC-MAY-21-ROR/kata-06-reloj-checador-k06-t03 into 22-checkin/checkout_function

yovanha reviewed (dismissed):
Good job!

OmarRoBra committed:
Fixig issues

YilKanda reviewed (approved):
LGTM!

AlejandroVI99 reviewed (approved):
nice!
